docs/current/sailbot_workspace/launch_files.md

```diff
@@ -0,0 +1,182 @@
+<iframe frameborder="0" style="width:100%;height:600px;" src="https://viewer.diagrams.net/?highlight=0000ff&nav=1&title=Global_Launch_Architecture.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D19_QAeRoTDw4qQRp6w6It4_d-8YmKw-ih%26export%3Ddownload"></iframe>
+
+The global launch process is intended to start up the entire system (both the development and production environments).
+This process invokes the package launch files for each ROS package used in the system through a global launch file.
+The process is as follows:
+
+1. The global launch file is invoked with the user passing arguments via the CLI and specifying a configuration file.
+2. Environment variables common to all ROS packages are declared. In addition, the global arguments common across
+all ROS packages are declared.
+3. For each package launch file:
+    - The CLI arguments, global argument declarations, and environment variables are passed into the package launch
+      file.
+    - Local arguments, specific to the package, are declared. Both the global and local arguments are parsed based on the
+      argument declarations and are set for use upon start up.
+    - The ROS nodes belonging to the package begin execution, utilizing the ROS parameters from the configuration file.
+
+## Invoking Launch Files
+
+??? tip "Stopping the execution of a launch file"
+    Entering ++ctrl+c++ in the terminal where the launch file was invoked will stop all associated
+    ROS packages from running.
+    > Use ++command+c++ for Mac OS
+
+### Package Launch
+
+At the bare minimum, the following packages need to be built with the `Build` or `Build All` VS Code task before launching:
+
+- `custom_interfaces`
+- The package you want to launch
+
+Once built, the package launch file can be invoked either in the CLI or using a VS Code command:
+
+=== ":octicons-command-palette-16: CLI"
+    Either the package and launch file name, or the path to the launch file can be used:
+
+    - **Method 1:** `ros2 launch <package> <launch file>`. This method can only be used when a launch file
+      is part of a built ROS package.
+    - **Method 2:** `ros2 launch <path to launch file>`. This method can be used regardless if a launch file
+      is in a ROS package or not.
+
+    !!! example "Launch via CLI Examples"
+        Let's launch local pathfinding using both CLI methods:
+
+        **Method 1**
+        ```sh
+        ros2 launch local_pathfinding main_launch.py
+        ```
+
+        **Method 2**
+        ```sh
+        ros2 launch $ROS_WORKSPACE/src/local_pathfinding/launch/main_launch.py
+        ```
+
+=== ":material-microsoft-visual-studio-code: VS Code"
+    Run the following VS Code command from the **Run and Debug** tab: `ROS: Launch (workspace)`
+
+    There will be a prompt to select which launch file to run. Select the desired launch file.
+
+### Global Launch
+
+Before running the system, be sure to run the `Build All` VS Code task to build all ROS packages.
```

Comment:
The `Build All` task is ran before the ros launch debug configuration, so no need to run it manually before